src/common/transformer_ctx.h

```diff
@@ -75,8 +76,8 @@ struct DecoderContext {
 
 public:
     DecoderContext(int _layers, int _hiddenSize, int _attHeadNum, int _kvHeadNum, int _imSize, const std::string &act,
-            float epsilon, int _vocabSize, int _embeddingSize, int _maxPositions, int _splitIdx, int _splits,
-            int numThreads = 0)
+            float epsilon, int _vocabSize, int _embeddingSize, int _maxPositions, int _maxPosEmbed, int _splitIdx,
```

Comment:
shall we reuse maxPositions as maxPosEmbed? so that we can make no change for DecoderContext.

Reply:
Added the explanations about those two variables.